supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -189,20 +223,36 @@ Always use "Administration Portal", including the capital P. When other departme
 [discrete]
 [[AMD64]]
 ==== image:images/yes.png[yes] AMD64 (noun)
-*Description*: _AMD64_ is the AMD implementation of a 64-bit version of the x86 architecture.
+*Description*: AMD's 64-bit version of the x86 architecture. Use this format for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, CLI objects. Use this format in general sentences when referring to OCP features. 
```

Comment:
AMD's 64-bit version of the x86 architecture. Use this format for OpenShift Container Platform (OCP) attributes, Kubernetes, Operators, APIs, CLI objects, and so on. Use this format in general sentences when referring to OCP features.